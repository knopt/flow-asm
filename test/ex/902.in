5
5
-6.508267124584659
-3.812587053212752 -3.8088617777605105 -8.877147882944403 -9.052624816315442 9.099801718203214 
4.24817713062729 4.959186061674989 -1.4102794803528251 6.3929495022686815 7.927322290930295 
8.826772794094069 -9.05495683951138 -0.28543284920553447 -9.630958249011531 0.7652590547683893 
1.2764499101758382 -6.360958659005924 6.551572964256067 -6.706371116071008 -6.063280375425386 
6.1402061444711435 3.2938084127911242 1.0176199679157598 -8.396865188562373 -7.395078208795249 

-0.9658776163419347 6.336495044573876 2.8874468891614384 6.687555224009042 -7.764931289006922 
