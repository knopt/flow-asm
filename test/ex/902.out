-140.85427639289506 -113.28677563914808 -180.50983920402987 -160.45702403563564 152.48836916309725 
56.11018620046117 189.7769601419617 63.77532577628125 404.83681284336933 281.6792531991825 
156.5497263540773 -387.99497963101453 25.02047807233861 -352.5986095455814 78.2568086124723 
-66.37568657523043 -281.57653032657305 293.87714932516354 -155.01541010076886 0.7548572606355739 
124.73088574723758 60.73400278217733 -1.7911061085470381 -177.95946501753534 -14.025166728099544 
